app/services/container_dashboard_service.rb

```diff
@@ -123,6 +123,7 @@ def heatmaps
     # Get latest hourly rollup for each node.
     node_ids = @ems.container_nodes if @ems.present?
     metrics = MetricRollup.latest_rollups(ContainerNode.name, node_ids)
+    metrics = metrics.where('timestamp > ?', 1.day.ago.utc).includes(:resource)
```

Comment:
I do like it here though. but that distinct and stuff... probably belongs where you put it (or a separate scope if you need the granularity for this)

by the way - that distinct and includes is pretty expensive, so if you don't really need - we may want to look into alternate solutions.